src/patternfly/components/Table/examples/Table.md

```diff
@@ -526,7 +526,7 @@ These classes can be used to ensure that the table changes between the tabular a
 {{#> table table--id="table-single-select-radio" table--IsGrid=true table--modifier="pf-m-grid-lg" table--attribute='aria-label="This is single select table with radio inputs"'}}
   {{#> table-thead}}
     {{#> table-tr}}
-      {{> table-cell-empty}}
+      {{> table-cell-empty table-cell--text='Radio select'}}
```

Comment:
Not really a blocker, just depends how strict we want to be about parity between Core and React examples. React uses "Row select" and similar verbiage for these types of empty cells.